### ISSUE TITLE
App status bar text color and navigation bar color of FilterViewController

### ISSUE THREAD

**Issue by jogendra** [fossasia/open-event-attendee-ios#63]
- Currently, status bar text color is **Black**. White color will look more good on the status bar since we are using **white** text color on all the navigation bar.
- Navigation bar background color on FilterViewController is white which is not matching to the Navigation bars background color on others.
<table>
<tr>
<td><img src="https://user-images.githubusercontent.com/20956124/39677948-4d0bd9dc-51a1-11e8-897e-b66212ba2648.png" width="250"></td>
<td><img src="https://user-images.githubusercontent.com/20956124/39677970-bedcd0fc-51a1-11e8-87b5-efc9b0935cff.png" width="250"></td>
</tr>
</table>

**Comment by DShivansh** [comment]
So basically i have to make the text color of the status bar white and the 
color of navigation bar on the filterViewController red

@imjog i am willing to do work on this

**Comment by imjog** [comment]
@DShivansh Yes, Please go ahead. Please change the status bar color of all the screens as well as filterViewController.

**Comment by baquer** [comment]
@imjog  i would like to take up this issue.
